src/scss/yellow/components/form/_steps.scss

```diff
@@ -23,6 +23,29 @@
     padding: $size-spacing-xxs;
     text-align: center;
 
+    &:nth-of-type(2) {
+      @media screen and (min-width: $media-small) {
+        flex: 2;
+      }
+    }
+
+    @media screen and (min-width: 360px) {
```

Comment:
We are breaking the concept of mobile-first here, the smallest screen has to be considered as default to write CSS rules and increase in the media-query as the resolution increases.